superset-frontend/src/features/databases/DatabaseModal/index.tsx

```diff
@@ -705,7 +721,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     setSSHTunnelPrivateKeys({});
     setSSHTunnelPrivateKeyPasswords({});
     setConfirmedOverwrite(false);
-    setUseSSHTunneling(false);
+    setUseSSHTunneling(undefined);
```

Comment:
why undefined instead of false?

Reply:
Oh I see, undefined means it was never checked and `false` means it was unchecked?